Sources/Widgets/Core/AbstractWidget/index.js

```diff
@@ -32,6 +32,7 @@ function vtkAbstractWidget(publicAPI, model) {
 
   publicAPI.deactivateAllHandles = () => {
     model.widgetState.deactivate();
+    model.activeState = null;
```

Comment:
`deactivateAllHandles()` being a convenient function and not the only way to deactivate handles, I wonder if a more robust fix wouldn't be : 
 - for the widget to observe the activeState, and if it becomes inactive,then set activeState to null.
 - or when deactivating a substate in the widget state, it could check if it is the same as `widget.getActiveState()` of the associated widget and if so, set it to null.

Reply:
Can the widget in focus be different than the `activeState` ?
if not, then deactivating the widget in focus could also make it loose focus.

Reply:
I certainly think so. An example would be a widget that processes mouse events while in focus but there is no handle state. A concrete example of this would be the PolyLineWidget, but there's no move handle.

The concern is mostly for existing widgets, e.g. a focused PolyLineWidget means you're drawing, but no active state means nothing happens. This is more of a "developer beware" note that I think would be good to document.